### DISCUSSION TITLE
Potential Multi-omics Tools and resources for new notebooks

### DISCUSSION THREAD

**Discussion by brynnz22** [microbiomedata/nmdc_notebooks#67] [General]
I talked to @Natalie-Winans and Juliane (one of PNNL's librarians) about multi-omics tools at PNNL we could potentially use. Natalie linked several tools, but pointed out they are not multi-omics integration tools but tools that can do analysis with multiple types of omics. These are:

- https://github.com/EMSL-Computing/ftmsRanalysis (mostly for metabolomics)
- https://github.com/EMSL-Computing/nmRanalysis (also mostly for metabolomics)
- https://github.com/pmartR/pmartR (handles proteomic, metabolomic, and lipidomic data)

For multi-omics, Natalie asked her friend who linked us to:
- https://www.emsl.pnnl.gov/science/instruments-resources/multiomics-analysis-portal
- http://mixomics.org/

Julianne linked me to the patents and software produced by DOE (VIPS - visual intellectual property search) that you can use to filter. I can look more into this. She mentioned that the downside is that a LOT of software doesn't make it into the DOE Code and thus into VIPS.
- VIPS: https://vips.pnnl.gov/search
 - DOE Code: https://www.osti.gov/doecode/

**Comment by kheal** [comment]
Awesome, thanks @brynnz22 .  Just FYI, nmRanalysis is for NMR-based metabolomics analyses, so I don't think it's applicable for the data we have in NMDC (yet!).

I couldn't get that first link to open :(.

pmartR and MAP (multiomics analysis portal) are the two I mentioned yesterday.  MAP is the gui/shiny interface with pmartR.  I went to a intro to MAP during techfest and have used pmartR a bit.  They both seem to be well supported, but don't do much integration between omics types as far as I can tell, though I could be wrong!

**Reply by brynnz22** [reply]
@kheal I updated the first link so it works.

**Comment by brynnz22** [comment]
@lamccue also mentioned looking into publications that use do multi-omics as inspiration for the next FY2025 notebook. I think we can use this discussion board as a dumping ground for any publications and tools we come across throughout the year leading up to the next notebook iteration. @SamuelPurvine, you were mentioned as someone who might have some familiarity with these. if anyone comes across any publications, tools, etc. that do multi-omics analyses can you drop them in here?
